### PR TITLE
static methods don't need final keyword, it's senseless

### PR DESCRIPTION
**Pull request by dk2k**
static methods don't need final keyword, it's senseless

### CODE REVIEW

File: components/camel-sql/src/generated/java/org/apache/camel/component/sql/stored/template/generated/TokenMgrError.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public class TokenMgrError extends Error
    * Replaces unprintable characters by their escaped (or unicode escaped)
    * equivalents in the given string
    */
-  protected static final String addEscapes(String str) {
+  protected static String addEscapes(String str) {
```
Comment (essobedo) [inline components/camel-sql/src/generated/java/org/apache/camel/component/sql/stored/template/generated/TokenMgrError.java:50]: This class is generated so I'm afraid that this change won't have any effect

**Comment by essobedo** [comment]
Please also use a different commit message, for something more clear

**Comment by essobedo** [comment]
BTW, maybe it is not really applicable here, but telling that it is meanless, it is not completely true, please read this https://stackoverflow.com/a/1743748/1997376

**Review by orpiske — CHANGES_REQUESTED** [review]
The commit message is not very clear, so requesting a change.

**Review by davsclaus — APPROVED** [review]

**Comment by dk2k** [comment]
the topic seems to be controversial. I'm OK with keeping this PR unmerged

**Comment by davsclaus** [comment]
merging anyway as generated class will re-gen

**Pull request merged**